opaleye.cabal

```diff
@@ -39,6 +39,7 @@ library
     , scientific          >= 0.3     && < 0.4
     , semigroups          >= 0.13    && < 0.21
     , text                >= 0.11    && < 2.1
+    , time
```

Comment:
I don't understand why `time` is needed.

Reply:
Removed.